### PR TITLE
CASMTRIAGE-3003 - Add step to diagnose and failover MDS

### PR DESCRIPTION
**Pull request by cdelatte-hpe**
## Summary and Scope

This adds additional troubleshooting documentation for Ceph filesystem/MDS

## Issues and Related PRs

* Resolves CASMTRIAGE-3003

### Tested on:

  * N/A

### Test description:

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)?
- Were continuous integration tests run? If not, why?
- Was upgrade tested? If not, why?
- Was downgrade tested? If not, why?
- Were new tests (or test issues/Jiras) created for this change?

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Troubleshooting Ceph MDS Client Connectivity Issues
+
+Use this procedure to diagnose and fix clients not logging into Ceph fs.
+
+**MOTE:** This section does not diagnose nor fix network issues.  Please ensure there that all networking if functional before proceeding.
```
Comment (bo-quan) [inline operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md:5]:
```suggestion
**MOTE:** This section does not diagnose nor fix network issues.  Please ensure that all networking is functional before proceeding.
```

File: operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+   cephfs_metadata  metadata  2403M  11.1T
+   cephfs_data      data    2641G  11.1T
+       STANDBY MDS
+   cephfs.ncn-s003.sjatdm
+   ```
+
+1. Fail over the MDS to trigger clients logins.
+
+   ```bash
+   ceph mds fail 0
```
Comment (bo-quan) [inline operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md:30]:
```suggestion
   # ceph mds fail 0
```

**Review by bo-quan — APPROVED** [review]

**Review by bo-quan — APPROVED** [review]

File: operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Troubleshooting Ceph MDS Client Connectivity Issues
+
+Use this procedure to diagnose and fix clients not logging into Ceph fs.
```
Comment (nrockershousen) [inline operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md:3]: Capitalize "fs"

File: operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+   cephfs.ncn-s003.sjatdm
+   ```
+
+1. Fail over the MDS to trigger clients logins.
+
+   ```bash
+   ceph mds fail 0
+   ```
+
+   NOTE: "0" refers to the active rank in our above output.
```
Comment (nrockershousen) [inline operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md:33]: Bold "NOTE:"

File: operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+**MOTE:** This section does not diagnose nor fix network issues.  Please ensure there that all networking if functional before proceeding.
+
+***IMPORTANT:*** The following commands can be run from ncn-m001/2/3 or ncn-s001/2/3. 
+
+## Procedure
+
+1. Identify if clients are not logged into Ceph FS.
+
+   ```bash
+   # ceph fs status
```
Comment (nrockershousen) [inline operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md:14]: In general, we are now separating input/output into separate code blocks. For example...

---

```
# ceph fs status
```

Example output:

```
   cephfs - 0 clients    <---- This indicates we have no clients connected
   ======
   RANK      STATE                MDS               ACTIVITY     DNS    INOS
   0        active      cephfs.ncn-s001.abiiiw  Reqs:    0 /s     0      0
   0-s   standby-replay  cephfs.ncn-s002.kyayma  Evts:   38 /s  35.5k  3220
        POOL         TYPE     USED  AVAIL
   cephfs_metadata  metadata  2403M  11.1T
   cephfs_data      data    2641G  11.1T
       STANDBY MDS
   cephfs.ncn-s003.sjatdm
   ```

File: operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
+# Troubleshooting Ceph MDS Client Connectivity Issues
```
Comment (nrockershousen) [inline operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md:1]: Change "Troubleshooting" to "Troubleshoot"

File: operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md (hunk 2 above)
Comment (cdelatte-hpe) [reply operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md:30]: Unless I give a complex example, I tend to not put the # in so they can utilize the copy function of github

**Review by nrockershousen — APPROVED** [review]

**Pull request merged**